apps/dashboard/app/views/dashboard/_pinned_apps.html.erb

```diff
@@ -1,3 +1,11 @@
 <h3><%= t('dashboard.pinned_apps_title') %> <small><%= t('dashboard.pinned_apps_caption_html', all_apps_url: apps_index_path) %></small></h3>
 
-<%= render(partial: "/dashboard/pinned_apps/group", collection: OodAppGroup.groups_for(apps: @pinned_apps, group_by: :original_category))  %>
+<%- group_by = Configuration.pinned_apps_group_by.to_sym -%>
+
+<%- if group_by.present? -%>
+  <%= render(partial: "/dashboard/pinned_apps/group", collection: OodAppGroup.groups_for(apps: @pinned_apps, group_by: group_by))  %>
```

Comment:
I think it improves readability in the views to avoid local variable assignment if possible. So

    <%- if Configuration.pinned_apps_group_by -%>
      <%= render(partial: "/dashboard/pinned_apps/group", collection: OodAppGroup.groups_for(apps: @pinned_apps, group_by: Configuration.pinned_apps_group_by))  %>